packages/webpack/src/buildtime/progress.js

```diff
@@ -81,9 +94,18 @@ module.exports = function progress({ steps }) {
     if (done.has(query)) {
       return
     }
-    throw Error(
-      `LavaMoatPlugin Plugin: Expected '${query}' to be done, but we're at '${steps[currentStep]}'`
+    reportError(
+      Error(
+        `LavaMoatPlugin Plugin: Expected '${query}' to be done, but we're at '${steps[currentStep]}'`
+      )
     )
   }
+  /**
+   * @param {Error[]} errors
+   */
+  API.reportErrorsTo = (errors) => {
+    errors.push(...compilationErrors)
+    compilationErrors = errors
```

Comment:
Are you explicitly intending to permutate the incoming `errors` here?
If not: 
```suggestion
    compilationErrors = [...errors, ...compilationErrors]
```

Reply:
I am. That's how compilation errors are aggregated in webpack - I have to push to `compilation.errors` to get them reported later instead of throwing (which prevents other compilation errors from getting surfaced to the user)